conference/react-js/src/reducers/forgon-password-error-message.js

```diff
@@ -0,0 +1,16 @@
+import actions from '../constants/actions-types';
+
+function forgotPasswordErrorMessage(isForgotPasswordSent = '', action) {
+  switch (action.type) {
```

Comment:
same. remove switch operator and create map object

Reply:
I am not sure I have understood correctly